OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -920,7 +920,7 @@ internal static class DreamProcNativeRoot {
         [DreamProcParameter("pixel_y", Type = DreamValueTypeFlag.Float)]
         public static DreamValue NativeProc_image(FastNativeProc.FastNativeProcBundle bundle, DreamObject? src, DreamObject? usr) {
             DreamObject imageObject = bundle.ObjectTree.CreateObject(bundle.ObjectTree.Image);
-            imageObject.InitSpawn(new DreamProcArguments(bundle.Arguments)); // TODO: Don't create another thread
```

Comment:
The reason this TODO was here was because I wanted `new /image()` to be included in the current thread's stack instead of starting a new one (and making errors more difficult to track down).

Is DreamObject initialization not included in threads anymore?

Reply:
/image has no New() and no DM-side Initialize() so it never spins up a thread at all.